presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -71,6 +71,26 @@ public boolean isOutputPartial()
         {
             return outputPartial;
         }
+
+        public static Step partialOutput(Step step)
+        {
+            if (step.isInputRaw()) {
+                return Step.PARTIAL;
+            }
+            else {
+                return Step.INTERMEDIATE;
```

Comment:
What is this `INTERMEDIATE` used for? It appears to only be referenced in some test code and an assertion, so it seems like dead code to me.